modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMessageBoards.testcase

```diff
@@ -174,4 +291,22 @@ definition {
 			workflowTask = "Update");
 	}
 
+	@description = "This is a test for LRQA-69694. Assert that the user can assign a workflow to Message Boards and verify that the process status be in Pending. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test WorkflowPendingForMessageBoard {
+
+		// TODO LRQA-69694 WorkflowPendingForMessageBoard pending implementation
+
+	}
+
+	@description = "This is a test for LRQA-69695. Assert that the user can assign a workflow to Message Boards and verify that the process status be in Untracked. "
+	@ignore = "Test Stub"
+	@priority = "3"
+	test WorkflowUntrackedForMessageBoard {
+
+		// TODO LRQA-69695 WorkflowUntrackedForMessageBoard pending implementation
+
+	}
+
```

Comment:
Hi Kristin! Could you, please, add stubs for this tests:
- description = "Verify that the user can edit an approved thread and it's status return to Pending"
priority = "3"
test = StatusReturnsToPendingWhenApprovedThreadIsEdited

- description = Verify that the user can not reply to a Pending Thread
priority = 3
test = CanNotReplyToPendingThread

- description = Verify that the reply to a Thread has workflow associated
priority = 4
test = ReplayHasWorkflow

- description = Verify that the user can Review the Thread through My Workflow Tasks
priority = 3
test = CanReviewThreadThroughMyWorkflowTasks

Reply:
Test stubs added!